tests/kotlin/test_files/.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.{kt,kts}]
+insert_final_newline=true
```

Comment:
no, something else.

Reply:
ktlint only supports `indent_size`, `continuation_indent_size`, `insert_final_newline`, and `max_line_length` editorconfig properties. Since none of these is specifically related to Kotlin I think we can remove this.

Reply:
So you have three other options :P 

`continuation_indent_size` is the one which is most specific to this linter.